packages/indexer/src/index.ts

```diff
@@ -26,13 +26,16 @@ export enum SequenceIndexerServices {
 
 const fetch = typeof global === 'object' ? global.fetch : window.fetch
 
-export class SequenceIndexerClient extends IndexerRpc {
+
+export class SequenceIndexer extends IndexerRpc {
   constructor(
     hostname: string,
-    public authorization?: {
-      jwtAuth?: string
-      accessKey?: string
-    }
+    public projectAccessKey?: string,
+    public jwtAuth?: string
+    // public authorization?: {
```

Comment:
perhaps using .. `new SequenceIndexer('https://polygon-indexer.sequence.app', { projectAccessKey: 'etcetc' })` is better dev experience, I believe so.